custom/rx_addendum.txt

```diff
@@ -1,3 +0,0 @@
-Medical Director of XXXX Clinics
```

Comment:
As discussed in above custom/readme_rx_printtofax.txt  , this is supported in the codebase.
@sunsetsystems 
Rod, do you think it makes sense to clean this thing out completely?